tests/attestation_plugin/plugin/tests.c

```diff
@@ -402,6 +402,26 @@ static void* _find_claim(
     return NULL;
 }
 
+static void _find_fmspc(uint8_t* tcb_info, uint8_t* fmspc, size_t fmspc_size)
+{
+    char* substr = "fmspc";
+    char* p_fmspc;
+    long data = 0;
+    char hex[2];
+
+    p_fmspc = strstr((char*)tcb_info, substr);
+    p_fmspc = p_fmspc + 8; // move pointer to start of value
+
+    for (size_t i = 0; i < fmspc_size; ++i)
+    {
+        hex[0] = *p_fmspc++;
+        hex[1] = *p_fmspc++;
+
+        data = strtol(hex, NULL, 16);
```

Comment:
`hex` is not null terminated. strtol will read past the buffer. You can fix that and retry.

Reply:
Thank you for pointing this out, this solved the problem.